assets/js/components/data.js

```diff
@@ -163,9 +163,12 @@ const dataAPI = {
 		const toRequest = [];
 		const deferredRequests = [];
 		const keyIndexesMap = {};
+		const noLowPriorityRequests = !! dataRequest.find( ( request ) => {
+			return request.priority < 10;
+		} );
```

Comment:
Personally I'd prefer a proper cast with `Boolean` here:

```js
const noLowPriorityRequests = Boolean( dataRequest.find( ( request ) => {
	return request.priority < 10;
} ) );
```

Doesn't seem to be used elsewhere in the code base though 🤷‍♂

Reply:
Yeah, I too like the clarity of the `Boolean` cast, but I see the `NOT NOT` approach in a lot of JS boolean casting in the wider JS community.

Colloquially I see the `!! foo` approach a lot in JS over `Boolean( foo )`, so I tend toward it. But that's obviously not Real Data™ 😄

I see `!! foo` elsewhere in the code already, so if we're to change that perhaps we should do it en-masse and create an eslint rule for it.

Reply:
[`no-implicit-coercion`](https://eslint.org/docs/rules/no-implicit-coercion) would be the rule for that. Perhaps one day 🙂